projects/plugins/jetpack/_inc/client/recommendations/feature-utils.js

```diff
@@ -435,7 +435,17 @@ export const getStepContent = ( state, stepSlug ) => {
 					'Congratulations, you’ve just unlocked comprehensive WordPress site security, including backups, malware scanning, and spam protection.',
 					'jetpack'
 				),
-				ctaText: __( 'Setup your new tools', 'jetpack' ),
+				ctaText: __( 'Set up your new tools', 'jetpack' ),
+				hasNoAction: true,
+			};
+		case 'welcome__starter':
+			return {
+				question: __( 'Welcome to Jetpack Starter!', 'jetpack' ),
+				description: __(
+					'Congratulations! You’ve unlocked essential security tools for your site including real-time backups and spam protection for comments and forms. Let’s get everything set up. It will only take a minute.',
```

Comment:
Nitpick, but I think we could include a comma here:

```suggestion
					'Congratulations! You’ve unlocked essential security tools for your site, including real-time backups and spam protection for comments and forms. Let’s get everything set up. It will only take a minute.',
```